### DISCUSSION TITLE
Compatibility Issues with PHP Version 8.0 or 8.1

### DISCUSSION THREAD

**Discussion by creativemouse** [elementor/elementor#20589] [Developers Community]
Has Elementor addressed any compatibility issues with updating the PHP version from 7.4 to 8.0? I am running an eCommerce website using the Hello theme, Elementor Pro, and WooCommerce. After updating the PHP version to 8.0 on my client's website, we received a website critical error message when navigating with the Product Categories. I did a search on this and found this item: https://github.com/elementor/elementor/issues/17322 which is exactly what I'm having issues with. I have downgraded back to 7.4 as I don't have the time to figure out compatibility issues with things I don't develop, particularly PHP.

**Comment by awebguy99** [comment]
Elementor better be compatible with PHP 8, **because 7.4 is now End of Life which means it might have security risks**
https://www.php.net/supported-versions.php

**Reply by kghoneycutt** [reply]
Elementor Hosting is running 7.4.33...

**Comment by nathanhill007** [comment]
I'm trying to update the elementor plugin and it's crashing my site.
GoDaddy says the plugin must not be compatible with PHP Version 8.0.27

**Reply by creativemouse** [reply]
Actually, Elementor does work with PHP V 8... on fresh installs for sure. I am still not able to update the PHP on sites that were built on Version 7.4. That has been my experience. I have no problem with new Elementor installations built on PHP 8.0. IMHO, there is just something preventing us from updating from 7.4 to 8.0. If Elementor can figure that out...I would be stoked!

**Comment by hitsgee** [comment]
I run 8.1 on most my sites and don't have issues. They did release a recent patch that claims to fix some PHP 8+ issues though.
